_posts/2026-07-09-log-detective-mcp.md

```
@@ -0,0 +1,77 @@
+
+## Using Log Detective MCP
+
+After installation, the MCP server and the tool is ready for use.
+
+Testing with Claude code, on a simple example of failed build from Copr,
+has revealed savings in the token budget, compared to the naive approach
+of the agent reading the log file directly.
+
+**Analysis without Log Detective MCP:**
+
+![](/assets/img/posts/log_analysis_without_log_detective_mcp.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)